.github/dependabot.yml

```diff
@@ -0,0 +1,181 @@
+version: 2
+updates:
+  # Root
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "root"
+      - "docker"
+    open-pull-requests-limit: 1
+    reviewers:
+      - "theopensystemslab/planx"
+
+  # Editor
+  - package-ecosystem: "npm"
```

Comment:
This should be set to `npm` when using pnpm

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/0d95e019-597e-4ada-9190-d6b48f2023fc)

Reply:
I hadn't caught this when updating the map repo today, opening a PR over there shortly to fix :+1:

Reply:
Yeah my linter flagged it locally and I ignored it initially as being outdated - kind weird!